home/import_assessments.py

```diff
@@ -398,15 +406,19 @@ def from_flat(cls, row: dict[str, str], row_num: int) -> "AssessmentRow":
             )
 
 
-def get_content_page_id_from_slug(slug: str) -> int:
+def get_content_page_id_from_slug(slug: str, locale: str) -> int:
     try:
-        page = ContentPage.objects.get(slug=slug)
+        page = ContentPage.objects.get(slug=slug, locale__language_code=locale)
     except ObjectDoesNotExist:
         raise ImportAssessmentException(
             f"You are trying to add an assessment, where one of the result pages "
-            f"references the content page with slug {slug} which does not exist. "
+            f"references the content page with slug {slug} and locale {locale} which does not exist. "
             "Please create the content page first."
         )
+    except ContentPage.MultipleObjectsReturned:
+        raise ImportAssessmentException(
+            f"Multiple ContentPages found with slug '{slug}' and locale '{locale}'. Please ensure slugs are unique within each locale."
+        )
```

Comment:
There's no test for this, but also it should be possible to have the same slug for two pages in the same locale, so I don't think we can create a test for this? If we can't create a test because it's not possible to get in this situation, then we can probably remove this?

Reply:
Yes, we can't create a test for this with this new fix so I'll remove it.